### PR TITLE
Add detail to the undergrad projects page

### PR DESCRIPTION
**Pull request by sajantanand**
Added project details and suggested readings.

### CODE REVIEW

File: members/anand/index.md [hunk 1]
```diff
@@ -2,4 +2,7 @@
 
 <!--# Publications -->
 
-<!-- \publist{https://arxiv.org/a/soejima_t_1.html} -->
\ No newline at end of file
+<!-- \publist{https://arxiv.org/a/soejima_t_1.html} -->
+
+\publist{https://arxiv.org/a/anand_s_1.html}
```
Comment (tomohiro-soejima) [inline members/anand/index.md:7]: sorry my README wasn't clear. you need either your profile page or your search result, but not both!

Comment (sajantanand) [reply members/anand/index.md:7]: Taige has both in his `index.md`, so I guess it doesn't break things.

Comment (tomohiro-soejima) [reply members/anand/index.md:7]: yes, it just duplicates your publication list.

Comment (tomohiro-soejima) [reply members/anand/index.md:7]: and Taige's `index.md` has already been corrected.

**Pull request merged**